wazo_amid/plugins/config/plugin.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2023 The Wazo Authors  (see the AUTHORS file)
```

Comment:
Wrong copyright

Reply:
Fixed in https://github.com/wazo-platform/wazo-amid/pull/89